src/main/resources/errors.yaml

```diff
@@ -160,6 +160,23 @@ request-errors:
       
       Resend using a supported Collection. 
 
+  # unscopped because this touches both the sort and the options
+  - scope:
+    code: MISSING_RERANK_QUERY_TEXT
+    title: Rerank query text is missing
+    body: |-
+      The findAndRerank command is missing the text to use as the query with the reranking model.
+
+      Reranking involves using a model to compare passages of text to a user query. 
+      
+      The query cnn be specified using either: 
```

Comment:
typo: cnn -> can

Reply:
wow, three typos in 10 lines a new record.